src/pages/DevPortalPage/DevPortalPage.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+import { REACT_APP_HASURA_URL } from '../../config/env';
+import { useCurrentUserAuthToken } from '../../hooks/useAuthToken';
+import { Box, Panel, Button } from '../../ui';
+
+const EXAMPLE_QUERY =
+  'query+MyCircles+%7B%0A++circles%28limit%3A+3%29+%7B%0A++++name%0A++++organization+%7B%0A++++++name%0A++++%7D%0A++++epochs%28limit%3A+3%2C+where%3A+%7Bended%3A+%7B_eq%3A+true%7D%7D%29+%7B%0A++++++id%0A++++++end_date%0A++++++start_date%0A++++%7D%0A++++users%28limit%3A+5%2C+order_by%3A+%7Bcreated_at%3A+asc%7D%29+%7B%0A++++++id%0A++++++name%0A++++++give_token_received%0A++++++give_token_remaining%0A++++++profile+%7B%0A++++++++twitter_username%0A++++++%7D%0A++++%7D%0A++%7D%0A%7D%0A';
+
+const getConsoleUrl = (authToken: string) => {
+  return `https://cloud.hasura.io/public/graphiql?endpoint=${REACT_APP_HASURA_URL}&header=Authorization:${encodeURIComponent(
+    `Bearer ${authToken}`
+  )}&query=${EXAMPLE_QUERY}`;
+};
```

Comment:
kind of a silly nit: would it make sense to direct the console url towards `http://localhost:8080` if the uri contains localhost? That way the process for community devs to test apis using an app-level auth token could utilize this interface.

If this is out of scope, I understand.

Reply:
The localhost:8080 Hasura instance console should be disabled / should not be used, for local dev people should use `yarn hasura console` so that any changes they make in the tables etc get persisted as yaml / sql in their local filesystem. 

If you want to use app level auth, you can just add `X-Hasura-Role` and even `X-Hasura-UserId` headers in the console and you can "act as" any specific user.

Reply:
wait, can you say more about this? i thought localhost:8080 and yarn hasura console were the same. when i make changes in the former, they show up in `yarn hasura metadata export`.

Reply:
ah, got it, thanks! will switch to that.